aptos-move/framework/src/natives/signature.rs

```diff
@@ -16,6 +17,38 @@ use move_deps::{
 use smallvec::smallvec;
 use std::{collections::VecDeque, convert::TryFrom};
 
+pub fn native_bls12381_public_key_validation(
+    _context: &mut NativeContext,
+    _ty_args: Vec<Type>,
+    mut arguments: VecDeque<Value>,
+) -> PartialVMResult<NativeResult> {
+    debug_assert!(_ty_args.is_empty());
+    debug_assert!(arguments.len() == 2);
+
+    // TODO: replace with proper gas cost
+    let cost = GasCost::new(super::cost::APTOS_LIB_TYPE_OF, 1).total();
```

Comment:
When will this be done?

Reply:
Victor is working on the overall gas story, so soon